admin_guide/overcommit.adoc

```diff
@@ -112,6 +112,26 @@ system tasks or daemons need more memory than was accounted for in the node's
 resource reservation. If a container specifies a limit on memory, it is
 immediately killed if it exceeds the limit amount.
 
+[[overcommit-ephemeral-storage]]
+=== Ephemeral Storage
+
+This applies only if you have enabled the Ephemeral Storage tech
+preview in {product-title} 3.10.  This is disabled by default; to
+enable this features, please see
+xref:../install_config/configuring_ephemeral.adoc[Configuring for
+Ephemeral Storage].
+
+A container is guaranteed the amount of ephemeral storage it
+requests. A container may use more ephemeral storage than requested,
+but once it exceeds its requested amount, it could be killed in if the
```

Comment:
s/killed/terminated

Reply:
Used the same wording as memory just above.

Reply:
Need to provide an explanation of ephemeral storage. We need to explain to the reader what ephemeral storage is, how it behaves, what types of ephemeral storage are available, what components (like containers) can use this type of storage, etc.